### PR TITLE
Update footprint

### PR DESCRIPTION
**Pull request by dyusuf**
Update footprint
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by dyusuf** [comment]
@BiocondaBot please add label

**Comment by bgruening** [comment]
@dyusuf https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/33258/workflows/4664b42d-ef95-445a-8abc-805587fe087a/jobs/124744

File: recipes/footprint/meta.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ source:
 
 build:
   skip: True  # [not linux]
+  noarch: generic
```
Comment (dyusuf) [inline recipes/footprint/meta.yaml:11]: noarch? what a magic

Comment (dyusuf) [reply recipes/footprint/meta.yaml:11]: the wraps different tools and R packages and only tested on Linux. noarch is not true here.

**Comment by dyusuf** [comment]
@bgruening https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/33268/workflows/1a629e9d-c3ff-41c7-b818-3c6eed28ccf4/jobs/124768

why linting insists on noarch? how is it so sure all the wrapped tools and packages can work cross-platform?

**Comment by dyusuf** [comment]
> @bgruening https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/33268/workflows/1a629e9d-c3ff-41c7-b818-3c6eed28ccf4/jobs/124768
> 
> why linting insists on noarch? how is it so sure all the wrapped tools and packages can work together cross-platform?

@aslihankarabacak you are the developer of footprint. do you think the tool can work across platforms?

**Comment by aslihankarabacak** [comment]
> > @bgruening https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/33268/workflows/1a629e9d-c3ff-41c7-b818-3c6eed28ccf4/jobs/124768
> > why linting insists on noarch? how is it so sure all the wrapped tools and packages can work together cross-platform?
> 
> @aslihankarabacak you are the developer of footprint. do you think the tool can work across platforms?

Hi @dyusuf it is only known to work on GNU/Linux and requires compiled non-portable tools. I hope that helps. Best, Asli

**Comment by dyusuf** [comment]
> > > @bgruening https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/33268/workflows/1a629e9d-c3ff-41c7-b818-3c6eed28ccf4/jobs/124768
> > > why linting insists on noarch? how is it so sure all the wrapped tools and packages can work together cross-platform?
> > 
> > 
> > @aslihankarabacak you are the developer of footprint. do you think the tool can work across platforms?
> 
> Hi @dyusuf it is only known to work on GNU/Linux and requires compiled non-portable tools. I hope that helps. Best, Asli

thanks @aslihankarabacak it helps. @bgruening I think the linting produces a false positive

**Comment by bgruening** [comment]
@dyusuf there is some documentation here: https://bioconda.github.io/contributor/guidelines.html

The build section is only for compiler and build-infrastructure like cmake etc.
What you want is the `host` section maybe. But if this tool only consists of perl and R packages, then this is a noarch package. As you probably don't need a package for all different R, perl and python versions that we support. If you think this is not true you can skip the linter, which I would not recommend. You can skip noarch packages for OSX - but if these are only scripts I don't see why this should not work, given that all dependencies are supposed to work on OSX.

**Comment by dpryan79** [comment]
@BiocondaBot please update

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**